modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/utils/numberShorten.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+/*
+ * If a number is bigger than 10000 it will transform into thousands (K).
+ * If a number is bigger than 1000000 it will transform into millions (M).
+ * If a number is bigger than 1000000000 it will transform into billions (B).
+ *
+ * 4 => 4
+ * 4000 => '4K'
+ * 4000000 => '4M'
+ * 4000000000 => '4B'
+ */
+export function numberShorten(value) {
```

Comment:
Did you try using [`formatStorage`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/format_storage.es.js#L17)

Reply:
One interesting distinction is that it has `denominator: 1024.0`... so either this or the other one might be slightly off 😉

Reply:
I didn't know that 🤔 I can configure the different suffix? 
`numberShorten` is used to represent the number with K (10.000 = 10K), M (1.000.000 = 1M), or B (1.000.000.000 = 1B). Can I get that with `formatStorage`? Thanks!

Reply:
[Tests will tell you that!](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_storage.es.js)

```javascript
	it('formats size under 1048575 bytes to kilobytes, with the default KB suffix', () => {
		expect(formatStorage(10400)).toEqual('10KB');
	});

	it('formats size 0 bytes to kilobytes, with the default KB suffix', () => {
		expect(formatStorage(0)).toEqual('0KB');
	});

	it('formats size over 1048575 bytes to megabytes, with the default MB suffix', () => {
		expect(formatStorage(1048576)).toEqual('1.0MB');
	});

	it('formats size over 1048575 bytes to megabytes with custom space, decimal separator, and suffix type parameters', () => {
		expect(
			formatStorage(1048576, {
				addSpaceBeforeSuffix: true,
				decimalSeparator: ',',
				suffixMB: 'megabytes',
			})
		).toEqual('1,0 megabytes');
	});
```

Reply:
I don't know if `formatStorage` is the perfect method for my use case... (see Y-Axis) 🤔 

<img width="1204" alt="Screenshot 2020-06-24 at 18 16 41" src="https://user-images.githubusercontent.com/15845466/85592989-ebe76d80-b646-11ea-81fb-c8808f934334.png">

I used:

```
Liferay.Util.formatStorage(payload.value, {
	denominator: 1000,
	suffixGB: 'B',
	suffixKB: 'K',
	suffixMB: 'M',
})
```
What do you think @jbalsas?

Reply:
So, these are views, so you're grouping per-1000 views? or something like that?

What's your mocked data for the `YAxis`? Based on your original screenshot, it should maybe be `denominator: 10`?

Reply:
The Y-Axis values are automatically calculated by Recharts, I am not adding it manually. I mocked the values of the bars:
```
const vocabularies = [
	{
		categories: [
			{
				key: 'education',
				name: 'Education',
				value: 478,
				vocabularyName: 'Stage',
			},
			{
				key: 'selection',
				name: 'Selection',
				value: 1055,
				vocabularyName: 'Stage',
			},
			{
				key: 'selection',
				name: 'Selection',
				value: 822,
				vocabularyName: 'Stage',
			},
		],
		key: 'business-decision-maker',
		name: 'Business Decision Maker',
		vocabularyName: 'Audience',
	},
	{ ... }
];
```

Reply:
That's what I mean, feels like you have `[500, 1000, 1500, 2000, 2500]` which `/1000` => `[0, 1, 1, 2, 2]`, so all is good there.

Reply:
But yeah, no need to worry about this if it's not clear...